### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by DariaNastas**
- [DEMO LINK](https://DariaNastas.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DariaNastas.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -15,8 +15,101 @@
...
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
```
Comment (GUSILLUS) [inline src/index.html:46]: Consider changing `nav_item` to `nav__item` to improve your BEM structure

File: src/index.html [hunk 2]
```diff
@@ -15,8 +15,101 @@
...
+            <a
+              class="link_nav is-active"
```
Comment (GUSILLUS) [inline src/index.html:48]: Same here `link_nav` to `nav__link`

File: src/index.html [hunk 3]
```diff
@@ -15,8 +15,101 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav_bloc">
```
Comment (GUSILLUS) [inline src/index.html:33]: Consider changing it from `nav_block` to `header`

File: src/index.html [hunk 4]
```diff
@@ -15,8 +15,101 @@
...
+      <a
+        href="#"
+        class="logo_icon"
```
Comment (GUSILLUS) [inline src/index.html:36]: `logo_icon` => `header__logo`

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ! Please fix your classnames logic and try to pass all your test in github

Feel free to ask help in your fe_chat

File: src/index.html (hunk 1 above)
Comment (DariaNastas) [reply src/index.html:46]: thanks, fixed classnames logic

**Comment by DariaNastas** [comment]
I actively use fe_chat. Mentors recommended to send for inspection. Yes, there are 4 failed tests BUT it's hard to tell what exactly is wrong, unfortunately….
Visually, everything is fine, the difference is almost not noticeable
Screenshot_39

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
GJ!
Let's improve your code

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,93 @@
...
+  transition: color 0.3s;
+}
+
+.is-active::after {
+  content: ' ';
+  position: absolute;
+  bottom: 0;
+  display: block;
+  background: #00acdc;
+  color: #00acdc;
```
[the remark this answers is not in the document]
Comment (DariaNastas) [reply src/style.css:67]: I'm sorry, I didn't understand what exactly I should do

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,82 @@
+html {
```
Comment (etojeDenys) [inline src/style.css:1]:
```
:root {
  --active-color: #00acdc;
}
```
declare the variable

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,82 @@
...
+  transition: color 0.3s;
+}
+
+.is-active::after {
+  content: ' ';
+  position: absolute;
+  bottom: 0;
+  display: block;
+  background: #00acdc;
+  color: #00acdc;
```
Comment (etojeDenys) [inline src/style.css:67]:
```suggestion
  background: var(--active-color);
  color: var(--active-color);
```
and reuse it everywhere

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 8]
```diff
@@ -15,8 +15,101 @@
...
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Samsung"
```
Comment (volodymyr-soltys97) [inline src/index.html:56]:
```suggestion
              href="#samsung"
```

File: src/index.html [hunk 9]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Smartphones"
```
Comment (volodymyr-soltys97) [inline src/index.html:64]:
```suggestion
              href="#smartphones"
```

File: src/index.html [hunk 10]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Laptops & Computers"
```
Comment (volodymyr-soltys97) [inline src/index.html:72]:
```suggestion
              href="#laptops&computers"
```

File: src/index.html [hunk 11]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Gadgets"
```
Comment (volodymyr-soltys97) [inline src/index.html:81]:
```suggestion
              href="#gadgets"
```

File: src/index.html [hunk 12]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Tablets"
```
Comment (volodymyr-soltys97) [inline src/index.html:89]:
```suggestion
              href="#tablets"
```

File: src/index.html [hunk 13]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Photo"
```
Comment (volodymyr-soltys97) [inline src/index.html:97]:
```suggestion
              href="#photo"
```

File: src/index.html [hunk 14]
```diff
@@ -15,8 +15,101 @@
...
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="Video"
```
Comment (volodymyr-soltys97) [inline src/index.html:105]:
```suggestion
              href="#video"
```

File: src/style.css [hunk 15]
```diff
@@ -1,3 +1,86 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+html:root {
```
Comment (volodymyr-soltys97) [inline src/style.css:6]:
```suggestion
:root {
```

File: src/style.css [hunk 16]
```diff
@@ -1,3 +1,86 @@
...
+  width: 100%;
+  height: 4px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:82]: Use a variable for this color

File: src/style.css [hunk 17]
```diff
@@ -1,3 +1,86 @@
...
+  display: block;
+  background: var(--active-color);
+  color: var(--active-color);
+  border-radius: 2px;
+  width: 100%;
+  height: 4px;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:78]: And here

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, you need to fix this problem after clicking on the links 
image
Also, don't forget to make a deploy again after your changes

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**